pugdebug/gui/stacktraces.py

```diff
@@ -9,11 +9,14 @@
 
 __author__ = "robertbasic"
 
+from PyQt5.QtCore import Qt, pyqtSignal
```

Comment:
Apparently it's a mistake. I think I used it for something which I removed, but forgot the import.